net/blockwise/blockwise.go

```diff
@@ -270,6 +270,7 @@ func newWriteRequestResponse[C Client](cc C, request *pool.Message) *responsewri
 	req.SetToken(request.Token())
 	req.ResetOptionsTo(request.Options())
 	req.SetBody(request.Body())
+	req.SetType(request.Type())
```

Comment:
We only want to set the type if it is either message.Confirmable or message.NonConfirmable. For other types, we prefer not to set them.

```suggestion
        if (request.Type() == message.Confirmable || request.Type() == message.NonConfirmable) {	 
          req.SetType(request.Type())
        }
```